presidio-anonymizer/README.MD

```diff
@@ -13,7 +13,9 @@ Presidio anonymizer comes by default with the following anonymizers:
   If "new_value" is not supplied or empty, default behavior will be: <entity_type>
   e.g: <PHONE_NUMBER>
 - Redact - removes the PII completely from text Parameters: None
-- Hash - hash the PII using sha256
+- Hash - hash the PII using either sha256, sha512 or md5.
+  Parameters:
+  * "hash_type" - sets the type of hashing. can be either sha256, sha512 or md5.
```

Comment:
Add that we're using sha256 by default if not passed

Reply:
done